solidity/contracts/script/ArbitrumDispatcher.s.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.0;
+
+import {Script} from "forge-std/Script.sol";
+import {TestRecipient} from "../../contracts/test/TestRecipient.sol";
+import {TypeCasts} from "../../contracts/libs/TypeCasts.sol";
+import {StandardHookMetadata} from "../../contracts/hooks/libs/StandardHookMetadata.sol";
+import {IMailbox} from "../../contracts/interfaces/IMailbox.sol";
+import {Message} from "../../contracts/libs/Message.sol";
+import {ArbitrumOrbitIsm} from "../../contracts/isms/hook/ArbitrumOrbitIsm.sol";
+import {ArbitrumOrbitHook} from "../../contracts/hooks/ArbitrumOrbitHook.sol";
+import {TestRecipient} from "../../contracts/test/TestRecipient.sol";
+
+/// @dev Given the hook and ISM are deployed, you can write their addresses as constants in this script,
```

Comment:
I am inclined not to add this type of forge script to the repo
we already have a test message CLI command that serves the same purpose

Reply:
(please remove this file)

Reply:
removed.